notebooks/image_models/solutions/object_detection_automl_vertex.ipynb

```diff
@@ -0,0 +1,427 @@
+{
```

Comment:
<ul><li>"Retreive" -&gt; "Retrieve"</li><li>It may be worth to just give a brief definition of the metric you look at in the next cell (i.e. "meanAveragePrecision") </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/220/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>